boards/silabs/dev_kits/xg24_ek2703a/Kconfig.defconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021, Sateesh Kotapati
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_XG24_EK2703A
+
+config CMU_HFXO_FREQ
```

Comment:
The HFXO and LFXO options no longer do anything, and should honestly be removed from all Series 2 boards, since this oscillators are now configured in device tree. However, this can be handled in a separate cleanup PR fixing all boards at once.

Reply:
Noted, thanks.